run_service.sh

```diff
@@ -175,6 +175,92 @@ get_on_chain_service_state() {
     echo "$state"
 }
 
+store=".trader_runner"
+rpc_path="$store/rpc.txt"
+operator_keys_file="$store/operator_keys.json"
+keys_json="keys.json"
+keys_json_path="$store/$keys_json"
+agent_address_path="$store/agent_address.txt"
+service_id_path="$store/service_id.txt"
+service_safe_address_path="$store/service_safe_address.txt"
+store_readme_path="$store/README.txt"
+
+# Function to create the .trader_runner storage
+create_storage() {
```

Comment:
Where is rpc stored in this function?

Reply:
It is stored outside the function:

https://github.com/valory-xyz/trader-quickstart/blob/0f99c967c2e7e8107c60b35f37b161a14fb5dce1/run_service.sh#L336

We do not want to clone the repo if the RPC does not support `eth_newFilter`.